src/components/page/Page.tsx

```diff
@@ -141,25 +164,25 @@ const Page = (props: { sketch: Sketch }) => {
     // ===== Keydown
     document.removeEventListener('keydown', eventHandlers.keydown);
     eventHandlers.keydown = (event: KeyboardEvent) => {
-      KeyboardHandler(state, draw, download)(event);
+      KeyboardHandler(state, loopState, draw, restart, download)(event);
     };
     document.addEventListener('keydown', eventHandlers.keydown, false);
   };
 
   const controlPanelUpdateHandler = (
     property: string,
     value: any,
-    updatedState: StringMap<any>,
+    // updatedState: StringMap<any>,

```

Comment:
I intentionally commented instead of removing this, just so I'm aware of what the imported library's interface is.... but this should be removed eventually.